### ISSUE TITLE
Add list of solutions I think are "great" to my profile page

### ISSUE THREAD

**Issue by marcoesposito1988** [exercism/discussions#149]
Hi there,

first of all, thanks a lot for your work on Exercism! I really like the idea and am getting quite addicted. And, learning a lot in the meantime.

However, I think there is some margin for improvement. Looking for a good, idiomatic solution after completing an exercise, which is the way I (at least) learn the most, is a nightmare. I read your discussions about improving this, and I hope that sorting solutions by the number of "This looks great!" votes will be possible, for example.

Still, I am really confused by the fact that there is no way to find the solutions which I marked as great. This should be trivial to implement, not shift the mechanics of the website and probably be really helpful. Sometimes I think about some solution which had a bright idea I can't recall any more, but it's basically impossible to find it again. 

Being able to follow a user (just have a list of people which I find good at some language) would also be great.

What do you think about it? (Sorry if this idea already popped up, I couldn't find it among the issues)

**Comment by iHiD** [comment]
@marcoesposito1988 Thanks for posting!  This will be a feature of the new version of exercism coming in the next couple of months :)

**Comment by kytrinyx** [comment]
@marcoesposito1988 You are absolutely right that there is room for improvement!

A few months ago we started digging into the design process that will let us put together a much better user experience. I wrote a little bit about it here: http://tinyletter.com/exercism/letters/the-delightful-design-of-exercism and will also be writing more about the things we've discovered and the questions we've found interesting (and perhaps surprising) answers to.

In the meanwhile, if someone wanted to add a paginated list of all the "looks great", filtered by language from the profile page, that would fine—it's just important to note that this would be a half-way measure, and that the current site will go away in not too long.
